src/main/java/at/fh/joanneum/irfc/rest/EventApi.java

```diff
@@ -0,0 +1,54 @@
+package at.fh.joanneum.irfc.rest;
+
+import at.fh.joanneum.irfc.model.event.EventDTO;
+import at.fh.joanneum.irfc.service.EventService;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+/**
+ * @author moe@softwaregaertner.at
+ **/
+@RequestScoped
+@Path("/events")
+public class EventApi {
+
+  @Inject
+  EventService eventService;
+
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  public List<EventDTO> getAll(){
+    return eventService.getAll();
+  }
+
+  @POST
+  @Produces(MediaType.APPLICATION_JSON)
+  @Consumes(MediaType.APPLICATION_JSON)
+  public EventDTO create(EventDTO eventDTOCreate){
```

Comment:
i think it looks cleaner if we call this eventDTO instead of eventDTOCreate, but if you like it this way its ok. 
if you change it here, change it everywhere

Reply:
same for eventDTOUpdate